### PR TITLE
[RHACS] [Docs] [4.5 and later versions] ROX-26541: Adding the dnsport option information

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.5+

Issue:
https://issues.redhat.com/browse/ROX-26541

Link to docs preview:
https://83378--ocpdocs-pr.netlify.app/openshift-acs/latest/cli/command-reference/roxctl-netpol.html#roxctl-netpol-generate_roxctl-netpol:~:text=%2D%2Ddnsport%20uint16,is%2053.

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.5`
    - `rhacs-docs-4.6`

### CODE REVIEW

File: modules/roxctl-netpol-generate.adoc [hunk 1]
```diff
@@ -21,6 +21,9 @@ $ roxctl netpol generate <folder_path> [flags] <1>
 |===
 |Option |Description
 
+|`--dnsport uint16`
```
Comment (adellape) [inline modules/roxctl-netpol-generate.adoc:24]: Nit:
```suggestion
|`--dnsport uint16`
```

**Review by adisos — APPROVED** [review]
Looks good

**Pull request merged**